api/environments/migrations/0037_add_uuid_field.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 3.2.23 on 2024-11-27 08:46
+
+from django.db import migrations, models
+import uuid
+
+from core.migration_helpers import AddDefaultUUIDs
+
+
+class Migration(migrations.Migration):
+    atomic = False  # avoid long running transaction
```

Comment:
Because we are not running all of these operations in a single transaction, there is a small probability that the last operation will fail. This could happen if the environment is created between step 2 and step 3 with the old code, which does not include the UUID.